modules/navier_stokes/include/fvkernels/PINSFVMomentumDiffusion.h

```diff
@@ -24,26 +25,24 @@ class PINSFVMomentumDiffusion : public FVFluxKernel
   ADReal computeQpResidual() override;
 
   /// the current element viscosity
-  const ADMaterialProperty<Real> & _mu_elem;
-  /// the neighbor element viscosity
-  const ADMaterialProperty<Real> & _mu_neighbor;
+  const Moose::Functor<ADReal> & _mu;
 
   /// the porosity
-  const VariableValue & _eps;
-  /// the neighbor element porosity
-  const VariableValue & _eps_neighbor;
+  const Moose::Functor<ADReal> & _eps;
 
   // Parameters for the gradient diffusion term
   /// Which momentum component this kernel applies to
   const int _index;
 
   /// Velocity as material properties
```

Comment:
```suggestion
  /// Velocity as functors
```
These are all over the headers in INSFV/PINSFV

I think the most important ones to change are the 'constant density' ones